lib/config/folderconfiguration.go

```diff
@@ -106,7 +106,7 @@ func (f FolderConfiguration) Versioner() versioner.Versioner {
 	}
 	versionerFactory, ok := versioner.Factories[f.Versioning.Type]
 	if !ok {
-		l.Fatalf("Requested versioning type %q that does not exist", f.Versioning.Type)
+		panic(fmt.Sprintf("Requested versioning type %q that does not exist", f.Versioning.Type))
```

Comment:
This was not printing stacktraces, now it will, which is a bit grim.

Reply:
I thought this could never happen anyway, thus panic. I could also just return an error and log a warning in the model and continue without a versioner.